.github/workflows/gradle.yml

```diff
@@ -2,9 +2,9 @@ name: SpringBoot CI with Gradle
 
 on:
   push:
-    branches: ["develop", "feature/validate-backend"]
+    branches: ["develop", "feature/validator"]
```

Comment:
Please don't update build file multiple times. It makes hard tracking bugs or errors especially like this PR which contains lots of file is changed.

You can use `feature/**` to match all the branches for this